Add trial details. (#3828)
platformsh/platformsh-docs: sites/friday/src/get-started/here/setup.md

{{% guides/requirements %}}

{{% note theme="info" title="Trials" %}}

> When you create an account on Upsun, you are also beginning a trial for your organization.

Reviewer: ```suggestion
When you create your first organization on Upsun, you are also activating your trial for that organization.
```

> When you create an account on Upsun, you are also beginning a trial for your organization.
> [See the documentation](/glossary#trial) for more details about limits to that trial.

Reviewer 2: ```suggestion
When you create an account on Upsun, you are also beginning a trial for your organization.
Get [more information on trials](/glossary#trial).
```
